### PR TITLE
Add event marker for *.twitter.com

### PR DESCRIPTION
**Pull request by monicachew**
Also add some code for setting aurora flags, but it totally doesn't work :(

### CODE REVIEW

**Comment by monicachew** [comment]
@mozkeeler ready for review

File: chartoptions.js [hunk 1]
```diff
@@ -96,13 +94,36 @@ var flags = {
...
+    },
   ]
 };
 
+var flags = {
+  type: 'flags',
+  showInLegend: false
+};
+
+
```
Comment (mozkeeler) [inline chartoptions.js]: nit: we probably don't need two blank lines here

**Comment by monicachew** [comment]
Hey, Garrett taught me something cool in the github UI! In the file diff view, you can click edit and fix the PR in place. I bet certain people would really find this useful in bugzilla :)

**Comment by mozkeeler** [comment]
Cool! I'll have to give that a try sometime.

**Pull request merged**